packages/office-ui-fabric-react/src/components/Calendar/Calendar.Props.ts

```diff
@@ -9,8 +12,10 @@ export interface ICalendar {
 export interface ICalendarProps extends React.Props<Calendar> {
   /**
   * Callback issued when a date is selected
+  * @param {Date} date - The date the user selected
```

Comment:
Can we omit the type from the jsdoc? It's already described by the typescript type.